src/test/suite/stubDemoTutorial.test.ts

```diff
@@ -77,7 +77,7 @@ suite('stub out a tutorial', () => {
 			await waitUntil(async () => {
 				await focusOnNamedTerminal(terminalName);
 				return terminalName === window.activeTerminal?.name;
-			}, 1000);
+			}, TERMINAL_WAIT_RETRY);
```

Comment:
a more reliable solution for the future might be to rewrite it using the new command "switch active terminal" https://issues.redhat.com/browse/FUSETOOLS2-1155?jql=project%20%3D%20FUSETOOLS2%20AND%20text%20~%20terminal